Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -310,7 +310,7 @@ class SettingsController: UITableViewController  {
         Settings.clear()
         
         // Stop calling wallet.bestBlockTimestamp() to update the best block age displayed on nav menu.
-        self.navigationMenuViewController()?.stopRefreshingBestBlockAge()
+//        self.navigationMenuViewController()?.stopRefreshingBestBlockAge()
```

Comment:
There should be a replacement for this.

Reply:
we are no longer showing the best block age in the nav menu so the timer was removed. out approach to showing the best block age has also changed. removed in 37e2453